[2021-05-18 16:38:34,377] ERROR: Exception on /login [POST] in /home/anton/PycharmProjects/epam_project/venv/lib/python3.8/site-packages/flask/app.py:1891
Traceback (most recent call last):
  File "/home/anton/PycharmProjects/epam_project/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/anton/PycharmProjects/epam_project/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/anton/PycharmProjects/epam_project/venv/lib/python3.8/site-packages/flask_restful/__init__.py", line 272, in error_router
    return original_handler(e)
  File "/home/anton/PycharmProjects/epam_project/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/anton/PycharmProjects/epam_project/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/anton/PycharmProjects/epam_project/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/anton/PycharmProjects/epam_project/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/anton/PycharmProjects/epam_project/src/views/autorisation_routes.py", line 27, in login
    employee = employee_schema.load(employee_json, session=db.session)
  File "/home/anton/PycharmProjects/epam_project/venv/lib/python3.8/site-packages/marshmallow_sqlalchemy/schema/load_instance_mixin.py", line 92, in load
    return super().load(data, **kwargs)
  File "/home/anton/PycharmProjects/epam_project/venv/lib/python3.8/site-packages/marshmallow/schema.py", line 725, in load
    return self._do_load(
  File "/home/anton/PycharmProjects/epam_project/venv/lib/python3.8/site-packages/marshmallow/schema.py", line 907, in _do_load
    raise exc
marshmallow.exceptions.ValidationError: {'position': ['Missing data for required field.'], 'email': ['Missing data for required field.'], 'first_name': ['Missing data for required field.'], 'last_name': ['Missing data for required field.'], 'message': ['Unknown field.']}
[2021-05-18 16:39:01,695] INFO: App startup in /home/anton/PycharmProjects/epam_project/src/__init__.py:27
[2021-05-18 16:39:01,882] INFO: App startup in /home/anton/PycharmProjects/epam_project/src/__init__.py:27
[2021-05-18 16:39:02,078] INFO: App startup in /home/anton/PycharmProjects/epam_project/src/__init__.py:27
[2021-05-18 16:39:02,150] INFO: App startup in /home/anton/PycharmProjects/epam_project/src/__init__.py:27
[2021-05-18 16:44:54,358] INFO: App startup in /home/anton/PycharmProjects/epam_project/src/__init__.py:27
[2021-05-18 16:44:54,744] INFO: App startup in /home/anton/PycharmProjects/epam_project/src/__init__.py:27
[2021-05-18 16:44:54,755] INFO: App startup in /home/anton/PycharmProjects/epam_project/src/__init__.py:27
[2021-05-18 16:44:55,036] INFO: App startup in /home/anton/PycharmProjects/epam_project/src/__init__.py:27
[2021-05-18 16:54:41,476] INFO: App startup in /home/anton/PycharmProjects/epam_project/src/__init__.py:27
[2021-05-18 16:54:41,478] INFO: App startup in /home/anton/PycharmProjects/epam_project/src/__init__.py:27
